### PR TITLE
fix readme h1 title in mgmt plane sdk

### PR DESCRIPTION
**Pull request by qiaozha**
fix https://github.com/Azure/azure-sdk-for-js/issues/22197
fix https://github.com/Azure/azure-sdk-for-js/issues/22203

### CODE REVIEW

**Comment by qiaozha** [comment]
@joheredi @sarangan12 Can you help take a look at this PR ? Thanks

File: src/generators/static/readmeFileGenerator.ts [hunk 1]
```diff
@@ -96,7 +96,14 @@ function createMetadata(
   const packageDirectoryName = names?.[1];
 
   const clientPackageName = packageDetails?.name;
-  const { clientClassName, serviceTitle } = getClientAndServiceName(codeModel.language, codeModel.info);
+  const { clientClassName, serviceTitle } = getClientAndServiceName(
+    codeModel.language,
+    codeModel.info
+  );
+  const simpleServiceNameTmp =
```
Comment (sarangan12) [inline src/generators/static/readmeFileGenerator.ts:103]: Can't we just name it `serviceName`/`simpleServiceName`/`heading` or anything without Tmp?

File: src/generators/static/readmeFileGenerator.ts [hunk 2]
```diff
@@ -82,7 +80,9 @@ function createMetadata(
   } = getAutorestOptions();
   const { addCredentials } = getSecurityInfoFromModel(codeModel.security);
 
-  const azureHuh = packageDetails?.scopeName === "azure" || packageDetails?.scopeName === "azure-rest";
+  const azureHuh =
```
Comment (sarangan12) [inline src/generators/static/readmeFileGenerator.ts:83]: what exactly is azure`Huh`? Can we use a meaningful name?

**Review by sarangan12 — APPROVED** [review]
Left 2 nit comments. Otherwise, I am fine with the changes

File: src/generators/static/readmeFileGenerator.ts (hunk 2 above)
Comment (qiaozha) [reply src/generators/static/readmeFileGenerator.ts:83]: azureHub : true means package names that are @azure or @azure-rest scope, in our smoke test, we are renaming all the packages with the  @msinternal scope. where the azureHub should be false.

Comment (qiaozha) [reply src/generators/static/readmeFileGenerator.ts:83]: Do you have any suggestions on a meaningful name ?

File: src/generators/static/readmeFileGenerator.ts (hunk 1 above)
Comment (qiaozha) [reply src/generators/static/readmeFileGenerator.ts]: changed

**Pull request merged**